api.go

```diff
@@ -338,7 +338,7 @@ func (c *Client) SetCustomTransport(customHTTPTransport http.RoundTripper) {
 	//           TLSClientConfig:    &tls.Config{RootCAs: pool},
 	//           DisableCompression: true,
 	//   }
-	//   api.SetTransport(tr)
+	//   api.SetCustomTransport(tr)
```

Comment:
Since we are here, shouldn't `SetCustomTransport` set `DisableCompression` to `true` unconditionally?

Reply:
Add 2018 to copyright header of this file.

Reply:
> Since we are here, shouldn't SetCustomTransport set DisableCompression to true unconditionally?

Not really since its custom we are not governing what a user wants to do.